pages/Stats.vue

```diff
@@ -0,0 +1,282 @@
+              <div class="container-fluid">
+                <div class="layout">
+                  <div
+                    class="flex fw-700 fs-15 my-2"
+                    style="font-weight: 600"
+                  >
+                    {{ item.nrNum }}
+                  </div>
+                </div>
+                <div
+                  v-for="(name, i) in item.names"
+                  :key="`names-layout-${i}`"
+                >
+                  <div
+                    class="flex fs-14"
+                    style="font-weight: 600"
+                  >
+                    {{ i + 1 }}. {{ name.name }}
+                    <span class="mx-2">
+                      <span :class="getClass(name.state)">{{ name.state }}
+                      </span>
+                    </span>
+                  </div>
+                  <div
+                    class="layout"
+                    style="font-size: 13px;">
+                    <div
+                      v-for="(text, j) in name.decision_text"
+                      :key="`decision-text-flex-${j}`"
+                      class="flex ft-ital mb-2 ml-4"
+                    >
+                      {{ text }}
+                    </div>
+                  </div>
+                </div>
+                <div
+                  v-if="item.comments.length > 0"
+                  class="ml-2 mt-1 dk-grey fs-14 fw-600"
+                >
+                  LAST COMMENT
+                </div>
+                <div
+                  v-if="item.comments.length > 0"
+                  class="dk-grey ml-2 mb-2"
+                >
+                  <div class="flex indent-10px">
+                    {{ item.comments[0].comment }}
+                  </div>
+                  <div class="flex ft-ital indent-10px">
+                    <template v-if="item.comments[0].examiner">
+                      <template v-for="(value, name) in item.comments[0].examiner">
+                        -{{ name }}: {{ value }},
+                      </template>
+                    </template>
+                    <template v-else>
+                      <span class="mr-2">–unknown examiner,</span>
+                    </template>
+                    {{ item.comments[0].timestamp }}
+                  </div>
+                </div>
+              </div>
+            </td>
+            <td
+              :class="getClass(item.stateCd)"
+              style="font-weight: 600"
+            >
+              {{ item.stateCd }}
+            </td>
+            <td>{{ item.lastUpdate }}</td>
+            <td>{{ item.activeUser }}</td>
+            <td>{{ item.furnished }}</td>
+          </tr>
+        </tbody>
+        <div
+          class="absolute bottom-1/2 right-1/2 translate-x-1/2 translate-y-1/2 transform"
+        >
+          <LoadingSpinner v-if="isLoading" />
+        </div>
+      </table>
+    </div>
+  </div>
+</template>
+
+<script>
```

Comment:
In other components, a TypeScript setup script using Vue 3's composition API is used. The `setup` function wouldn't be required. You may have to type the variables using the types in [the types file](https://github.com/bcgov/name-examination/blob/1c56690a0a783a0e199c565d243245b21609dab6/types/index.ts#L238) to solve any errors.
```suggestion
<script setup lang="ts">
```

Reply:
I think using setup() in stats is fine. It's hard to say which one is better right now because the function is quite simple.

Reply:
They both work, it's just that it would be consistent with the rest of the components in the application. You wouldn't need the return statement at the end of the script either.

<img width="552" alt="image" src="https://github.com/bcgov/name-examination/assets/144159881/748673c3-6283-478d-8d9e-94ba43433cac">

Also, since the stats page works, perhaps I can approve this PR and the rest of the requested changes can be moved to a future refactor PR instead maybe.